JenkinsfileCloudfrontInvalidation

```diff
@@ -0,0 +1,19 @@
+node('linux && docker') {
```

Comment:
Why create a dedicated jenkins file? Why not add a stage to the production jenkins file?